src/main.rs

```diff
@@ -53,6 +53,12 @@ async fn main() -> Result<(), Box<dyn std::error::Error>>
                         .default_value(std::env::var("RUSTFLAGS").unwrap_or(String::new()))
                         .allow_hyphen_values(true)
                         .help("Flags to send to rustc"),
+                    Arg::new("submit")
+                        .short('S')
+                        .long("submit")
+                        .required(false)
+                        .help("Submit answer")
+                        .action(clap::ArgAction::SetTrue),
```

Comment:
Very nice, however, maybe add a conflict with the test flag?
https://docs.rs/clap/latest/clap/struct.Arg.html#method.conflicts_with

Reply:
Didn't know that was a thing, but that totally makes sense. Great catch!